CHANGELOG.md

```diff
@@ -12,9 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Documented how to use layouts (see [issue 63](https://github.com/KNowledgeOnWebScale/walder/issues/63))
 - Supporting comunica lenient querying (see [issue 68](https://github.com/KNowledgeOnWebScale/walder/issues/68))
 - FrontMatter metadata available in layout templates (see [issue 72](https://github.com/KNowledgeOnWebScale/walder/issues/72))
+- Logging to HtmlConverter
+- Default support for Nunjucks templates
```

Comment:
Only change I see is in package.json. Is that enough?

Reply:
Yes that is enough. Because the `consolidate` library loads it dynamically when needed.